test/fixtures/lodash-fp-mix/expected.js

```diff
@@ -1,34 +1,34 @@
 'use strict';
 
-var _lodashCollectionMap2 = require('lodash/collection/map');
+var _convert = require('lodash-fp/convert');
 
-var _lodashCollectionMap3 = _interopRequireDefault(_lodashCollectionMap2);
+var _convert2 = _interopRequireDefault(_convert);
 
-var _lodashMathAdd2 = require('lodash/math/add');
+var _reject = require('lodash/collection/reject');
 
-var _lodashMathAdd3 = _interopRequireDefault(_lodashMathAdd2);
+var _reject2 = _interopRequireDefault(_reject);
 
-var _lodashArrayTake2 = require('lodash/array/take');
+var _take = require('lodash/array/take');
 
-var _lodashArrayTake3 = _interopRequireDefault(_lodashArrayTake2);
+var _take2 = _interopRequireDefault(_take);
 
-var _lodashCollectionReject2 = require('lodash/collection/reject');
+var _add = require('lodash/math/add');
 
-var _lodashCollectionReject3 = _interopRequireDefault(_lodashCollectionReject2);
+var _add2 = _interopRequireDefault(_add);
 
-var _lodashFpConvert2 = require('lodash-fp/convert');
+var _map = require('lodash/collection/map');
 
-var _lodashFpConvert3 = _interopRequireDefault(_lodashFpConvert2);
+var _map2 = _interopRequireDefault(_map);
 
-var _lodashFp = (0, _lodashFpConvert3['default'])({
-  'map': _lodashCollectionMap3['default'],
-  'add': _lodashMathAdd3['default'],
-  'take': _lodashArrayTake3['default'],
-  'reject': _lodashCollectionReject3['default']
-});
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }
+const _lodashFp = (0, _convert2.default)({
```

Comment:
Not sure why _lodashFp is not processed, I change the variableDeclaration kind to 'var' now.